homeassistant/components/tado/sensor.py

```diff
@@ -124,6 +126,21 @@ def get_geofencing_mode(data) -> str:
         attributes_fn=lambda data: {"time": data["weatherState"]["timestamp"]},
         data_category=SENSOR_DATA_CATEGORY_WEATHER,
     ),
+    TadoSensorEntityDescription(
+        key="freshness",
+        translation_key="freshness",
+        state_fn=lambda data: data["freshness"]["value"],
+        data_category=SENSOR_DATA_CATEGORY_AIRCOMFORT,
+    ),
+    TadoSensorEntityDescription(
+        key="last open window",
+        translation_key="last_open_window",
+        state_fn=lambda data: datetime.strptime(
```

Comment:
Also, I think this is the same as datetime.from_iso or something along those lines